src/current/_includes/v23.2/backups/serverless-locality-aware.md

```diff
@@ -0,0 +1 @@
+CockroachDB {{ site.data.products.serverless }} clusters operate with a [different architecture]({% link cockroachcloud/architecture.md %}#cockroachdb-serverless) compared to CockroachDB {{ site.data.products.core }} and CockroachDB {{ site.data.products.dedicated }} clusters. These architectural differences have implications for how locality-aware backups can run. Serverless clusters will scale resources depending on whether they are actively in use, which means that it is less likely to have a SQL pod available in every locality. As a result, Serverless clusters are more likely to have ranges that do not match with any of the cluster's localities, which can lead to more ranges backed up to a storage bucket in a different locality. You should consider this as you plan a backup strategy that must comply with [data domiciling]({% link v23.2/data-domiciling.md %}) requirements.
```

Comment:
Would this fit better in Cloud documentation? Does "Serverless" apply to self-hosted at all? (I don't think it does, but this is all subtle. Perhaps Steven will correct me.)

Reply:
Yes, I have add it to the unsupported features page to date. But, right now the information architecture for backup/restore in cloud needs some work (which I am working on the Breather Week). I will add this include to that work when there is a good space on the new Cloud docs I am working on.